add: Ext PG integration to Flink docs (#113)
aiven/aiven-docs: docs/products/flink/howto/ext-pg-flink-integration.md

    --endpoint-name <external-pg-endpoint-name> \
    --endpoint-type external_postgresql \
    --user-config-json '{"host":"<postgres_host>", \
    "port":"<postgres_port>", "db_name":"<postgres_db>", \
    "user":"<postgres_user>", "password":"<postgres_password>", \
    "ssl":"<ssl_mode>", "sslrootcert":"<path_to_ssl_cert>"}'
```

Parameters:

- `--project <project-name>`: The name of your Aiven project.
- `--endpoint-name <external-pg-endpoint-name>`: The name for the new endpoint.
- `--endpoint-type external_postgresql`: The type of endpoint to create.
- `--user-config-json`: Configuration details of your PostgreSQL database in JSON format.

### Verify integration with service

After creating the integration between Aiven for Apache Flink and your external
PostgreSQL database, verify that the
integration has been created successfully, and create applications that
use the integration.

```bash
  avn service integration-list --project <project-name> <flink-service-name>
```

Parameters:
- `--project <project-name>`: The name of your Aiven project.
- `<flink-service-name>`: The name of your Aiven for Apache Flink service.

> To create Aiven for Apache Flink applications, you will need the
> integration ID of the Aiven for Apache Flink service. Obtain the
> `integration_id` from the integration list.

Reviewer: Let's skip the future to shorten things? 

```suggestion
To create Aiven for Apache Flink applications, retrieve
the integration IDof the Aiven for Apache Flink
service from the integration list.
```

Author: Move this line to the next section and updated the content.

### Create Aiven for Apache Flink applications

> With the integration ID obtained, you're now ready to develop Aiven for Apache Flink
> applications that utilize the data from your external PostgreSQL database
> as either a source or a sink.

Reviewer: We've already mentioned this at the top of the page and previous section, let's limit repetition?

```suggestion
```

For information on how to create Aiven for Apache Flink applications, see
[avn service flink create-application](/docs/tools/cli/service/flink#avn%20service%20flink%20create-application).

## Configure integration using Aiven Console

Integrating your Aiven for Apache Flink service with an external PostgreSQL
database can be achieved through the Aiven Console by following these steps:

1. Log in to [Aiven Console](https://console.aiven.io/) and choose your project.
1. From the **Services page**, you can either
   - Create a new [**Aiven for Apache Flink service](/docs/platform/howto/create_new_service)

>    - Select an existing service

Reviewer: I think all the substeps of this whole list are all sequential?

1. Configure an external PostgreSQL database integration endpoint:

>    - Go to the **Projects** screen where all services are listed.

Reviewer: Is this the **Services** sidebar entry?
